template/bootstrap3/bin-shop-basket.html

```diff
@@ -3,6 +3,26 @@
     <i class="fa fa-shopping-cart fa-fw"></i>
     ({{$ctrl.preview.quantity}}) <span ng-if="$ctrl.mode == 'link'">{{$ctrl.preview.presentablePrice}}</span>
 </a>
+<a ng-click="$ctrl.onDropdownClick()" ng-if="$ctrl.mode == 'dropdown-link' && $ctrl.preview.quantity > 0">
```

Comment:
Careful here, if you inspect the actually rendered DOM you'll notice that your <div class="basket-items-dropdown" is moved to the same level as <a ng-click="$ctrl.onDropdownClick()". HTML5 allows nesting div's inside anchors buf you cannot nest anchors then  the browser (Chrome) starts behaving tricky.